README.md

```diff
@@ -6,9 +6,18 @@ A starting place for FOIA in the US government.
 
 This project is currently working on getting people to the right place in the government to file their FOIA request.
 
-Some related repos:
+## Outside Contributors
 
-* [foia](https://github.com/18F/foia) - Discussion forum and miscellaneous resources and scrapers.
+Hello! If you're interested in learning more about this project, check out some related repos and don't be afraid to ask us questions (general questions usually go here: [foia](https://github.com/18F/foia)). 
+
+If you'd like to contribute to our project, please check out our [foia-hub] (https://github.com/18F/foia-hub) repo. We try to tag things that are easy to pick up without being entrenched in our project with a ["help wanted"] (https://github.com/18F/foia-hub/labels/help%20wanted%21) tag. Things in our [backlog] (https://github.com/18F/foia-hub/milestones/Backlog) are usually also up for grabs, so let us know if you'd like to pick something up from there. 
```

Comment:
I don't think things in our backlog are up for grabs as it were. The backlog isn't sorted on Github and that means that it's unclear which issue would next provide the most value. We put things in the backlog that are intended to be worked on in the future, or that we may consider in the future. I think for now, we should stick to outside contributors participating through the 'help wanted' tags. 

The other 'tag' they could help with is the 'bug' tag. As that's code that's already there - but not working as we'd like.